renderdoc/driver/metal/metal_device.h

```diff
@@ -72,6 +72,20 @@ class WrappedMTLDevice : public WrappedMTLObject
                                           newRenderPipelineStateWithDescriptor,
                                           RDMTL::RenderPipelineDescriptor &descriptor,
                                           NS::Error **error);
+  DECLARE_FUNCTION_WITH_RETURN_SERIALISED(WrappedMTLComputePipelineState *,
+                                          newComputePipelineStateWithDescriptor,
+                                          RDMTL::ComputePipelineDescriptor &descriptor,
+                                          MTL::PipelineOption options,
+                                          MTL::AutoreleasedComputePipelineReflection *reflection,
+                                          NS::Error **error);
+  DECLARE_FUNCTION_WITH_RETURN_SERIALISED(WrappedMTLComputePipelineState *,
```

Comment:
The signature is wrong here : these APIs need to take a WrappedMTLFunction* not a MTL::Function*
then the WrappedMTLFunction * will need to be serialised